apps/wallet/src/shared/utils/index.ts

```diff
@@ -22,6 +23,14 @@ export function usePageView() {
         customRPC && apiEnv === 'customRPC' ? customRPC : apiEnv.toUpperCase();
 
     useEffect(() => {
+        // NOTE: This is a hack to work around hook timing issues with the Growthbook SDK.
+        // Issue: https://github.com/growthbook/growthbook/issues/915
+        setTimeout(() => {
```

Comment:
@Jibz1 I know you have a similar change to this in another PR, but this version has a bugfix to work around the described issue as well.

Reply:
I will remove mine